test/space_calculation_test.rb

```diff
@@ -112,6 +112,8 @@ def expect_to_execute(command)
     end
   end
 
+# storage-ng
+=begin
```

Comment:
Yes, they are unusual and that's exactly why are we using it to comment temporary commented code. This format makes very easy to find the temporary patches.
```ruby
  # storage-ng
  value = "replaced value because of storage-ng"
=begin
  value = "something"
  value << "original that we need to comment"
  value << "and that ends here"
=end
```

These comments are not expected to be there after the whole rewrite. That's what I want them to look different.